protocol/communities/community_changes.go

```diff
@@ -73,176 +73,3 @@ func (c *CommunityChanges) HasMemberLeft(identity string) bool {
 	_, ok := c.MembersRemoved[identity]
 	return ok
 }
-
-func EvaluateCommunityChanges(origin, modified *protobuf.CommunityDescription) *CommunityChanges {
```

Comment:
@mprakhov I am wondering why it was removed from here and moved back to community.go?

Reply:
From my understanding it was the same fn as I had, but I wrapped a little bit more code inside

Reply:
Restored already with: https://github.com/status-im/status-go/pull/3765

Reply:
yes, I saw and approved that PR :)